slangpy/tests/device/test_external_imgui.py

```
@@ -0,0 +1,78 @@
+    )
+
+    imgui.new_frame()
+    imgui.set_next_window_pos((4, 4))
+    imgui.set_next_window_size((140, 90))
+    imgui.begin("external imgui")
+    imgui.text("Hello from imgui_bundle")
+    imgui.image(imgui.ImTextureRef(ui_context.texture_id(checker_texture)), (24, 24))
+    imgui.end()
+    imgui.render()
+    draw_data = imgui.get_draw_data()
+    synced_textures = sync_draw_data_textures(device, ui_context, draw_data)
```

Ruff (0.15.5)

[error] Local variable `synced_textures` is assigned to but never used

Remove assignment to unused variable `synced_textures`

(F841)